CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+* **New Resource:** `google_bigtable_logical_view` ([#9876](https://github.com/hashicorp/terraform-provider-google-beta/pull/9876))
+* **New Resource:** `google_bigtable_materialized_view` ([#9862](https://github.com/hashicorp/terraform-provider-google-beta/pull/9862))
+* **New Resource:** `google_os_config_v2_policy_orchestrator_for_folder` ([#9841](https://github.com/hashicorp/terraform-provider-google-beta/pull/9841))
+
+IMPROVEMENTS:
+* beyondcorp: Added `upstreams` fields to `google_beyondcorp_application` resource ([#9890](https://github.com/hashicorp/terraform-provider-google-beta/pull/9890))
+* compute: Added fields like `raw_key`, `rsa_encrypted_key`, `kms_key_service_account` to all relevant resources on `google_compute_instance_template` and `google_compute_region_instance_template` ([#9880](https://github.com/hashicorp/terraform-provider-google-beta/pull/9880))
+* compute: added `disk_id` to `google_compute_region_disk` resource ([#9855](https://github.com/hashicorp/terraform-provider-google-beta/pull/9855))
+* compute: marked `location` field as required in `google_compute_interconnect` resource ([#9865](https://github.com/hashicorp/terraform-provider-google-beta/pull/9865))
+* container: added `data_cache_count` to `ephemeral_storage_local_ssd_config` for `google_container_node_pool` ([#9851](https://github.com/hashicorp/terraform-provider-google-beta/pull/9851))
+* container: added update for `gvnic` to `google_container_node_pool` ([#9834](https://github.com/hashicorp/terraform-provider-google-beta/pull/9834))
+* dataplex: added `notification_report` field to `google_dataplex_datascan` resource ([#9857](https://github.com/hashicorp/terraform-provider-google-beta/pull/9857))
+* dns: added `target_name_servers.domainName` to resource`google_dns_managed_zone ` (beta) ([#9832](https://github.com/hashicorp/terraform-provider-google-beta/pull/9832))
+* gkehub: added `configmanagement.config_sync.deployment_overrides` field to `google_gke_hub_feature_membership` resource ([#9828](https://github.com/hashicorp/terraform-provider-google-beta/pull/9828))
+* identityplatform: Added `response_type` field to `google_identity_platform_oauth_idp_config` ([#9856](https://github.com/hashicorp/terraform-provider-google-beta/pull/9856))
+* netapp: added `custom_performance_enabled`, `total_throughput_mibps`, and `total_iops` fields to `google_netapp_storage_pool` resource (beta) ([#9872](https://github.com/hashicorp/terraform-provider-google-beta/pull/9872))
+* networkservices: added `metadata` field to `google_networkservices_lbtrafficextension` resource ([#9849](https://github.com/hashicorp/terraform-provider-google-beta/pull/9849))
+* secretmanager: added `tags` field to `google_secret_manager_secret` to allow setting tags for secrets at creation time ([#9850](https://github.com/hashicorp/terraform-provider-google-beta/pull/9850))
```

Comment:
Remove when revert is done

Reply:
FYI, revert was just done with https://github.com/GoogleCloudPlatform/magic-modules/pull/13786